cmd/containerd/command/service_windows.go

```diff
@@ -337,6 +337,13 @@ func initPanicFile(path string) error {
 	// and replace it.
 	if st.Size() > 0 {
 		panicFile.Close()
+
+		if content, err := ioutil.ReadFile(path); err != nil {
+			logrus.WithError(err).Debug("failed to read containerd panic.log")
+		} else {
+			logrus.WithField("panic.log", string(content)).Debug("containerd panic.log content")
```

Comment:
Most of the time this will be empty, seems like we should only log if there's content.

Reply:
there's a `st.Size() >0` check already, so I think we're fine here.

Reply:
How big can this file be?
Can we get a the `Writer` for this and `io.Copy` instead of reading the entire file in?
https://pkg.go.dev/github.com/sirupsen/logrus?utm_source=godoc#Entry.WriterLevel

Reply:
I'm unsure dumping the content of the file is the right fallback behavior here. FYI, there is an old unmerged PR #4787 from @kevpar, that shows sending a long string could be problematic on Windows.

Reply:
> there's a st.Size() > 0 check already, so I think we're fine here.

Oop you're right

Reply:
> How big can this file be?

Largest I've seen was around 300KB. 

@kzys, oh, I forgot about that PR. yes, I agree this might be a problem.

we could truncate the content, like we do for shim panic logs currently.

Reply:
Would you want the end or beginning of the log? Or the middle around some specific keyword?